src/Basque/Mews.Fiscalizations.Basque/DtoToModelConverter.cs

```diff
@@ -28,11 +29,45 @@ internal static class DtoToModelConverter
         );
     }
 
+    public static SendInvoiceResponse Convert(
+        LROEPJ240FacturasEmitidasConSGAltaRespuesta response,
+        string qrCodeUri,
+        string xmlRequestContent,
+        string xmlResponseContent,
+        String1To100 signatureValue
+        )
+    {
+        var invoiceIdentifier = response.Registros.Single().Identificador.Item as IDFacturaType;
+        var invoiceState = response.Registros.Single().SituacionRegistro;
+        return new SendInvoiceResponse(
+            xmlRequestContent: xmlRequestContent,
+            xmlResponseContent: xmlResponseContent,
+            qrCodeUri: qrCodeUri,
+            tbaiIdentifier: invoiceIdentifier.NumFactura,
+            received: DateTime.ParseExact(invoiceIdentifier.FechaExpedicionFactura, "dd-MM-yyyy", CultureInfo.InvariantCulture),
+            state: ConvertBizkaiaState(invoiceState.EstadoRegistro),
+            description: invoiceState.DescripcionErrorRegistroES,
+            stateExplanation: string.Empty,
+            signatureValue: signatureValue,
+            csv: string.Empty,
+            validationResults: response.Registros?.Select(v => Convert(v.SituacionRegistro))
+        );
+    }
+
     private static SendInvoiceValidationResult Convert(Dto.ResultadosValidacion validation)
     {
         return new SendInvoiceValidationResult(ParseEnum<ErrorCode>(validation.Codigo), validation.Descripcion, validation.Azalpena);
     }
 
+    private static SendInvoiceValidationResult Convert(SituacionRegistroType situacionRegistro)
+    {
+        return new SendInvoiceValidationResult(
+            errorCode: ConvertBizkaiaErrorCodes(situacionRegistro.CodigoErrorRegistro), 
+            description: situacionRegistro.DescripcionErrorRegistroEU, 
```

Comment:
Above you're using DescripcionErrorRegistroES which i believe is the description in Spanish
and here you're using DescripcionErrorRegistroEU which i believe is in Basque

I would stick with one language 😄